src/app/view/components/dot-navigation/components/dot-nav-item/dot-nav-item.component.scss

```diff
@@ -5,93 +5,92 @@
 :host {
     display: block;
 
-    ::ng-deep {
-        .ui-tooltip {
-            &.dot-nav__tooltip {
-                margin-left: $basic-padding + $navigation-width-collased - $navigation-width;
+    &.dot-nav-item__collapsed {
+        position: relative;
+        transition: background-color $basic-speed ease;
 
-                .ui-tooltip-text {
-                    overflow-wrap: break-word;
-                    min-width: 100px;
-                    max-width: 150px;
-                    text-align: center;
-                }
+        &:hover {
+            background-color: rgba(var(--color-white_rgb), 0.1);
+
+            dot-sub-nav {
+                height: auto !important;
```

Comment:
is `important` needed?

Reply:
Yes, because the angular animation set a inline height